lib/Smokeping/probes/IRTT.pm

```diff
@@ -60,12 +68,12 @@ DOC
 }
 
 sub new ($$$) {
-    my $proto = shift;
-    my $class = ref($proto) || $proto;
-    my $self = $class->SUPER::new(@_);
+	my $proto = shift;
+	my $class = ref($proto) || $proto;
+	my $self = $class->SUPER::new(@_);
```

Comment:
super already does what you do on lines 63 and 64

Reply:
Sorry, I haven't figured this one out, I need to get $class with those two lines to call SUPER, right? :)

Reply:
you can do:

```perl
sub new {
    my $self = shift->SUPER::new(@_);
    ...
}
```

Reply:
Ah, I see, it works and I pushed it. This seems to be the first probe that does it that way, but it's good that it's simpler. :)